ui/component/walletSendTip/view.jsx

```diff
@@ -246,16 +246,16 @@ export default function WalletSendTip(props: Props) {
 
   React.useEffect(() => {
     if (!hasSelected && hasSelectedTab && activeTab !== hasSelectedTab) {
-      setActiveTab(hasSelectedTab);
+      setActiveTab(claimIsMine ? TAB_BOOST : hasSelectedTab);
       setSelected(true);
     }
-  }, [activeTab, hasSelected, hasSelectedTab, setActiveTab]);
+  }, [activeTab, claimIsMine, hasSelected, hasSelectedTab, setActiveTab]);
 
   React.useEffect(() => {
     if (!hasSelectedTab && activeTab !== hasSelectedTab) {
-      setPersistentTab(activeTab);
+      setPersistentTab(claimIsMine ? TAB_BOOST : activeTab);
     }
-  }, [activeTab, hasSelectedTab, setPersistentTab]);
+  }, [activeTab, claimIsMine, hasSelectedTab, setPersistentTab]);
```

Comment:
- [ ] Something's incomplete ... maybe the button label needs update too?

<img width="470" alt="image" src="https://user-images.githubusercontent.com/64950861/177526788-605b3b1b-0118-498e-a536-78f65b560864.png">

Reply:
still happens? I don't see it

Reply:
Yes.  Doesn't happen in production, so feels like something related to this change.
- Uploads > Click "Support" from the context menu of a Post